x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/CleanupShrinkIndexStepTests.java

```diff
@@ -72,8 +72,8 @@ public void testPerformActionDoesntFailIfShrinkingIndexNameIsMissing() {
         CleanupShrinkIndexStep cleanupShrinkIndexStep = createRandomInstance();
         cleanupShrinkIndexStep.performAction(indexMetadata, clusterState, null, new ActionListener<>() {
             @Override
-            public void onResponse(Boolean complete) {
-                assertThat(complete, is(true));
+            public void onResponse(Void unused) {
+                assertTrue("onResponse must be called", true);
```

Comment:
I don't think we need this assert since it's basically a noop now (`assert true`)? Should we just add a `fail("reason");` to the `onFailure` handler below?